CMake/macros.cmake

```diff
@@ -120,6 +120,11 @@ FUNCTION( RUN_QMC_APP_NO_COPY TESTNAME WORKDIR PROCS THREADS TEST_ADDED ${ARGN}
             MESSAGE("Disabling test ${TESTNAME} (building without MPI)")
         ENDIF()
     ENDIF()
+    IF ( TEST_ADDED )
+       SET(TEST_LABELS "")
+       EXECUTE_PROCESS(COMMAND ${PROJECT_SOURCE_DIR}/tests/scripts/test_labels.py ${TESTNAME} ${QMC_CUDA} ${ENABLE_SOA} ${QMC_COMPLEX} ${QMC_MIXED_PRECISION} OUTPUT_VARIABLE TEST_LABELS)
```

Comment:
The RESULT_VARIABLE could be checked to ensure the script returns success.  When the stdout returned from the script is used to set the labels, there should be a guard involving the exit code.

Reply:
That's a good suggestion.  How do you access the error code?

Reply:
using the RESULT_VARIABLE parameter

Reply:
Sorry, was conflating with OUTPUT_VARIABLE.  Will be straightforward.

Reply:
Better now?